### ISSUE TITLE
Allow excludes to be passed in a file

### ISSUE THREAD

**Issue by joshkunz** [joshkunz/ashuffle#113]
From #112:

> being able to specify excludes in a file that ashuffle reads from would be helpful, i'm sure at some point i'm going to overflow the max command length with this shellscript 😁

`ashuffle` should probably allow users to specify excludes in a file if they have a large number of exclusion rules.

**Comment by joshkunz** [comment]
I started writing a custom parser for these rules, but the need for quoting made the parser complicated enough that I figured it would be better to just pick up an off-the-shelf structured config language instead. I'm thinking I'll go with a YAML config along the lines of:

```yaml
rules:
  - artist: "foo"
    album: "bar"
  - album: "foo"
```

I think YAML is a bit easier to write, and it's a super-set of JSON, so shell scripts can generate the rules pretty easily as well:

```bash
echo '{ "rules": ['
echo "{ \"artist\": \"${ARTIST}\", \"album\": \"${ALBUM}\"},"
echo ']}'
```

**Comment by joshkunz** [comment]
This has been released as `v3.13.0`. cc @ryan-sg

**Comment by ryan-sg** [comment]
thanks @joshkunz !  rewrote all my excludes in yaml form and works like a charm :+1:
